pkg/agent/clean/cluster.go

```diff
@@ -349,6 +354,29 @@ func cleanupRoles(client *kubernetes.Clientset) []error {
 	return errs
 }
 
+func cleanupWebhookResources(client *kubernetes.Clientset) []error {
+	logrus.Info("Starting cleanup of webhook-specific resources")
+	var errs []error
+
+	if !dryRun {
+		err := client.RbacV1().ClusterRoleBindings().Delete(context.TODO(), "rancher-webhook", metav1.DeleteOptions{})
```

Comment:
Nit:

Generally not a fan of `context.TODO()` in production code. You can use `context.Background()` for the same effect. Marking this as a nit since:
- Other parts of this file use todo
- todo and background are technically the same.

Reply:
I don't like TODOs either, but I have recently been more accepting of them based on the last part of the function documentation. 
> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).